lib/imagemounter/index.js

```diff
@@ -0,0 +1,119 @@
+import { readFile, existsSync, lstatSync, createReadStream } from 'fs';
+import { BaseServicePlist } from '../base-service';
+import B from 'bluebird';
+import log from '../logger';
+
+const MOBILE_IMAGE_MOUNTER_SERVICE_NAME = 'com.apple.mobile.mobile_image_mounter';
+
+class ImageMounter extends BaseServicePlist {
+
+    _checkReturnError(ret) {
+        if (ret.Error) {
+            throw new Error(ret.Error);
+        }
+    }
+
+    /**
+     * Lookup for mounted images.
+     * * @param {string?} imageType Type of image, `Developer` by default.
```

Comment:
extra star

Reply:
what are other image types?

Reply:
> what are other image types?

To be honest, I don't know. It's just a parameter exposed by protocol. So I left this just in case there are other types